frontend/src/Nav/Nav.js

```diff
@@ -12,14 +12,16 @@ const BtnContainer = styled.div`
 	flex-flow: row;
 `;
 
-const Nav = () => {
+const Nav = props => {
 	return (
 		<NavBar>
 			<TextIMG alt={'TeamHome banner'} src={textLogo} />
 			<BtnContainer>
-				<StyledLink to="/home">Home</StyledLink>
 				<StyledLink to="/dashboard">Dashboard</StyledLink>
 				<StyledLink to="/settings">Settings</StyledLink>
+				<StyledLink to="/" onClick={props.handleLogout}>
```

Comment:
If this is an extension of React routers Link or NavLink component, you're gonna want to use the `exact` property on it to keep this from trying to load multiple routes.